static/index.html

```diff
@@ -5,48 +5,47 @@
   <meta charset='utf-8'>
   <title>Remote Play</title>
   <meta name="viewport" content="user-scalable=no, width=device-width, initial-scale=1, maximum-scale=1">
-  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bulma@0.9.2/css/bulma.min.css">
-  <link rel="stylesheet" href="static/style.css">
+  <!-- <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bulma@0.9.2/css/bulma.min.css"> -->
```

Comment:
Since you have commented out Bulma, looks like we can indeed get rid of all the Bulma classes.

Reply:
"When I press any button on mobile, it gets a highlight color (the .button:hover CSS), and that stays until I press another button. The color should only flash when I press the button but should not sustain." 

![Screen Shot 2021-04-14 at 12 57 57 AM](https://user-images.githubusercontent.com/78054396/114667894-47a9e280-9cbd-11eb-898a-2678542fa3f4.png)

The background color is hover so not sustain. But are you talking about this blue border of the current selected button?

Reply:
On Android, the button itself stays blue. I guess these blogposts summarizes the same issue https://medium.com/@mezoistvan/finally-a-css-only-solution-to-hover-on-touchscreens-c498af39c31c  https://css-tricks.com/solving-sticky-hover-states-with-media-hover-hover/